package.json

```diff
@@ -85,8 +85,7 @@
     "trailingComma": "es5"
   },
   "publishConfig": {
-    "access": "public",
-    "tag": "next"
```

Comment:
oops shall I revert this change? I think we discussed that we want to remove it?

Reply:
Yes. You should revert it and it will publish on `@next` then you can use `npm dist-tag` to move it to `@latest`

Reply:
Ah yes I remember. I feel I’ll forget it though, I’d rather publish to latest and if something goes wrong set the latest dist-tag to the previous version and fix it with another release ...